sfy-buoy/sfy-artemis/src/main.rs

```diff
@@ -3,7 +3,7 @@
 #![no_main]
 
 #[cfg(not(feature = "deploy"))]
-use panic_probe as _;
+use panic_probe as _; 
```

Comment:
Preferably don't add trailing whitespaces. You can use `cargo fmt` to auto-format the code.